src/reducers-actions/orders.ts

```diff
@@ -2,7 +2,12 @@ import { Actions } from 'reducers-actions'
 import { AuctionElement } from 'api/exchange/ExchangeApi'
 import { ZERO } from '@gnosis.pm/dex-js'
 
-export type ActionTypes = 'OVERWRITE_ORDERS' | 'APPEND_ORDERS' | 'UPDATE_ORDERS' | 'UPDATE_OFFSET'
```

Comment:
Why you no like types?
Can we keep 'em types?

Reply:
I prefer being able to use the types directly as shown inside one of the other commits. But I guess with TS it's the same? Makes it easier anyways inside Middleware for example

Reply:
I do get auto-completes for types as much as everything else, as long as the editor knows what I'm talking about. Or is this a different context where the type is not known?

Reply:
How do you grab a ActionTypes when it's a conditional type?  Was my problem. What is your preference over the previous implementation?

Reply:
Oh boy. So much I don't understand going on here :sweat_smile: 

I'd prefer to keep them as types because it's less verbose, but I won't block you from changing to Enum if that's necessary.

Reply:
Can use `ActionTypes.APPEND_ORDERS` directly instead of `string`